package.json

```diff
@@ -35,6 +35,8 @@
 	},
 	"type": "module",
 	"dependencies": {
-		"spatext": "^0.3.0-beta.20230312"
+		"@codemirror/lang-xml": "^6.0.2",
+		"spatext": "^0.3.0-beta.20230313",
+		"svelte-codemirror-editor": "^1.1.0"
```

Comment:
Updated spatext to fix a bug I found there, and added https://codemirror.net/ for prettier typesetting of code.